### ISSUE TITLE
Accessibility: The aria-expanded attribute should be rendered on the ComboboxInput

### ISSUE THREAD

**Issue by lebowska** [tailwindlabs/headlessui#1604]
**What package within Headless UI are you using?**

@headlessui/vue

**What version of that package are you using?**

For example: v1.6.4

**What browser are you using?**

Chrome

**Reproduction URL**

Your Vue Combobox (Autocomplete) demo at https://headlessui.dev/vue/combobox

**Describe your issue**

The "aria-expanded" attribute should be rendered on the ComboboxInput too. I see that it's correctly rendered on the ComboboxButton, but being the ComboboxInput a role of type "combobox", it sould always requires the "aria-expanded" attribute, as stated here: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role
"A WAI-ARIA combobox only has one attribute that is required that authors specify: [aria-expanded]"

I also see that the React version correctly renders the attribute on both the button and the input, so I think it could be a bug on the Vue version of the input component.

Thank for your hard work

![Screenshot 2022-06-18 160528](https://user-images.githubusercontent.com/6949697/174442055-f9c2a769-8c3a-4df2-9963-30eb1da5a40f.png)
